src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/app_start/routing/web/WebWorkflowRouteConfig.kt

```diff
@@ -10,27 +10,28 @@ object WebWorkflowRouteConfig : RouteConfig
     private val runReports = setOf("*/reports.run")
 
     override val endpoints: List<EndpointDefinition> = listOf(
-        WebEndpoint("/workflow", WorkflowRunController::class, "createWorkflowRun", HttpMethod.post)
-            .json()
-            .secure(runReports),
-        WebEndpoint("/workflows", WorkflowRunController::class, "getWorkflowRunSummaries")
-            .json()
-            .secure(runReports)
-            .transform(),
-        WebEndpoint("/workflows/summary", WorkflowRunController::class, "getWorkflowRunSummary", HttpMethod.post)
-            .json()
-            .secure(runReports),
-        WebEndpoint("/workflows/:key", WorkflowRunController::class, "getWorkflowRunDetails")
-            .json()
-            .secure(runReports)
-            .transform(),
-        WebEndpoint("/workflows/:key/status", WorkflowRunController::class, "getWorkflowRunStatus")
-            .json()
-            .secure(runReports)
-            .transform(),
-        WebEndpoint("/workflow/validate", WorkflowRunController::class, "validateWorkflow", HttpMethod.post)
-            .json()
-            .secure(runReports)
-            .transform()
+            WebEndpoint("/workflow", WorkflowRunController::class, "createWorkflowRun", HttpMethod.post)
+                    .json()
+                    .secure(runReports),
+            WebEndpoint("/workflows", WorkflowRunController::class, "getWorkflowRunSummaries")
+                    .json()
+                    .secure(runReports)
+                    .transform(),
+            WebEndpoint("/workflows/summary", WorkflowRunController::class, "getWorkflowSummary", HttpMethod.post)
```

Comment:
I guess this rename is to distinguish this from the now renamed WorkflowRunSummary type in the frontend.

Reply:
Yes, seemed a little clearer to me given the return types